apps/question-bank/src/test/java/gov/cdc/nbs/questionbank/addsection/AddSectionServiceTest.java

```diff
@@ -0,0 +1,45 @@
+package gov.cdc.nbs.questionbank.addsection;
+
+import gov.cdc.nbs.questionbank.addsection.exception.AddSectionException;
+import gov.cdc.nbs.questionbank.addsection.model.CreateSectionRequest;
+import gov.cdc.nbs.questionbank.addtab.repository.WaUiMetaDataRepository;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@ExtendWith(MockitoExtension.class)
+class AddSectionServiceTest {
+
+    @InjectMocks
+    private CreateSectionService createSectionService;
+
+    @Mock
+    private WaUiMetaDataRepository waUiMetaDataRepository;
+
+    @Test
+    void createSectionServiceTest() throws AddSectionException {
+
+        CreateSectionRequest createSectionRequest =
+                new CreateSectionRequest(10L, 10L, "Local", "T");
+
+        CreateSectionResponse createSectionResponse =
+                createSectionService.createSection(123L, createSectionRequest);
+        assertEquals("Section Created Successfully", createSectionResponse.message());
+    }
+
+    @Test
+    void createSectionServiceTestException() throws AddSectionException {
+
+        AddSectionException exception =
+                assertThrows(AddSectionException.class, () -> createSectionService.createSection(123L, null));
+
+        assertEquals(
+                "gov.cdc.nbs.questionbank.addsection.exception.AddSectionException: Add section exception",
+                exception.toString());
```

Comment:
the assertEquals is redundant. The assertThrows will cause the test to fail if the exception isn't thrown.